tools/firmware.toit

```diff
@@ -648,6 +652,14 @@ extract-cmd -> cli.Command:
         cli.OptionEnum "format" ["binary", "elf", "ubjson", "image", "qemu", "tar"]
             --help="Set the output format."
             --default="binary",
+        cli.Option "partition-table"
+            --help="Override the partition table of the envelope."
+            --type="file",
+        cli.OptionPatterns "partition"
```

Comment:
Maybe change this to add-partition or something and keep partition as an alias?

Reply:
I will leave it as is for now. Will discuss this with you.